src/Temporalio/Exceptions/WorkflowQueryRejectedException.cs

```diff
@@ -0,0 +1,25 @@
+using Temporalio.Api.Enums.V1;
+
+namespace Temporalio.Exceptions
+{
+    /// <summary>
+    /// Thrown when a query is rejected by the worker due to bad workflow status.
```

Comment:
Worth explicitly distinguishing the difference between this and `WorkflowQueryFailedException` in the docstring I think

Reply:
Will do! Also noticed a typo here that may help explain the difference too

```suggestion
    /// Thrown when a query is rejected by the server due to bad workflow status.
```